.github/workflows/ci-core.yml

```diff
@@ -91,24 +96,37 @@ jobs:
               - '!nix-darwin-shell-hook.sh'
               - '!LICENSE'
               - '!.github/**'
-    
+
+      - name: Resolve affected files to affected modules
+        if: ${{ steps.match-every.outputs.non-ignored == 'true' }}
+        id: resolved-modules
+        shell: bash
+        run: |
+          bash ./.github/scripts/map-affected-files-to-modules.sh '${{ steps.match-every.outputs.non-ignored_files }}'
```

Comment:
So I see why the above was changed, however the `non-ignored` files filter list is an optimization for running ci-core. If we make changes to `integration-tests/*` these don't affect unit tests and therefore we don't need to run this test suite.